pir/cpp/client_test.cpp

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright 2020 the authors listed in CONTRIBUTORS.md
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "client.h"
+
+#include "gtest/gtest.h"
+
+namespace pir {
+namespace {
+
+class PIRClientTest : public ::testing::Test {
+ protected:
+  void SetUp() {
+    client_ = PIRClient::Create().ValueOrDie();
+    ASSERT_TRUE(client_ != nullptr);
+  }
+
+  std::unique_ptr<PIRClient> client_;
+};
+
+TEST_F(PIRClientTest, TestSanity) {
+  constexpr std::size_t dbSize = 1000;
+
+  auto desiredIndex = std::rand() % dbSize;
```

Comment:
Let's avoid random values in unit tests. Pick a random value once and hardcode it. That avoids heisenbugs. We can have separate tests for monte carlo testing, but not in the base unit tests.